templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -319,6 +319,16 @@
             <td {$valueStyle}>{$form.payment_instrument_id.html} {help id="payment_instrument_id"}</td>
             </td>
           </tr>
+          {if $form.card_type}
+            <tr id="cardType" class="crm-contribution-form-block-card_type">
+              <td class="label">{$form.card_type.label}</td>
+              <td {$valueStyle}>{$form.card_type.html}</td>
+            </tr>
+            <tr id="cardNumber" class="crm-contribution-form-block-credit_card_number">
```

Comment:
OK - I see what you are doing - giving the same to these fields as payment form fields have in order to trick the js into handling them both.  There must be a less tricksy way to do it